app/controllers/books_controller.rb

```diff
@@ -0,0 +1,56 @@
+class BooksController < ApplicationController
+  skip_before_filter :verify_authenticity_token, only: [:destroy]
```

Comment:
Hi @phiggins, sorry for all the questions.  I was trying to verify the book belongs to the current_member in order to delete it.  For some reason, skipping this was the only way for current_member to be available to the destroy action.